app/Database/CourseQueries.hs

```diff
@@ -109,11 +118,12 @@ getDescriptionD (Just key) = do
     maybeDistribution <- get key
     return $ fmap distributionDescription maybeDistribution
 
--- | Queries the database for all times corresponding to a given meeting.
-buildMeetTimes :: Entity Meeting -> SqlPersistM Tables.MeetTimes
+-- | Queries the database for all times corresponding to a given meeting and construct a MeetTime
```

Comment:
This documentation change isn't necessary. We rely on the type signature to indicate that a `MeetTime` is returned.